### ISSUE TITLE
WebServiceQuerier & post_max_size

### ISSUE THREAD

**Issue by MidgeS** [structureddynamics/OSF-Web-Services-PHP-API#3]
Hi,
i just came across a small oversight which can cause quite some hassle.

Inside your WebServiceQuerier you have this check:

``` php
151:        $uploadMaxFileSize = ini_get("post_max_size");
152:        $uploadMaxFileSize = str_replace("M", "000000", $uploadMaxFileSize);
153:
154:        if($uploadMaxFileSize > strlen($this->parameters))
```

So the problem i had appeared when i set post_max_size to 0. Which as statet in the [manual](http://www.php.net/manual/en/ini.core.php#ini.post-max-size) should be unlimited file size. This caused me some trouble since your code would never allow data to be send.
Further you are replacing M but not the also possible K and G.

This is how i altered the Code to prevent further problems

``` php
        $uploadMaxFileSize = ini_get("post_max_size");
        $uploadMaxFileSize = str_replace("M", "000000", $uploadMaxFileSize);
        $uploadMaxFileSize = str_replace("G", "000000000", $uploadMaxFileSize);
        $uploadMaxFileSize = str_replace("K", "000", $uploadMaxFileSize);

        if($uploadMaxFileSize > strlen($this->parameters) || $uploadMaxFileSize == 0)
```

[0001-fixed-post_max_size.patch.zip](https://github.com/structureddynamics/OSF-Web-Services-PHP-API/files/491292/0001-fixed-post_max_size.patch.zip)

**Comment by fgiasson** [comment]
Yes I agree that this is an issue. Would it be possible for you to fork the OSF repository, then to commit your changes and do a pull-request? I will review an integrate from you (and you will get the credit along the way).

Thanks!
